README-MALAY.md

```diff
@@ -0,0 +1,142 @@
+- [Why Open Source and How?](https://careerkarma.com/blog/open-source-projects-for-beginners/) karya [@james-gallagher](https://careerkarma.com/blog/author/jamesgallagher/)
+- [How to get started with Open Source - By Sayan Chowdhury](https://www.hackerearth.com/getstarted-opensource/)
+- [What open-source should I contribute to](https://kentcdodds.com/blog/what-open-source-project-should-i-contribute-to/) karya Kent C. Dodds
+
+## Menggunakan Pengawalan Versi
+- [Think Like (a) Git](http://think-like-a-git.net/) - Pengenalan Git untuk "pemula maju", tetapi masih berusaha, untuk memberi anda strategi mudah untuk bereksperimen dengan selamat dengan git.
+- [Try Git](https://try.github.io/) - Belajar Git dalam 15 minit dalam laman anda.
+- [Everyday Git](https://git-scm.com/docs/giteveryday) - Satu set perintah minimum yang berguna untuk Everyday Git.
+- [Oh shit, git!](https://ohshitgit.com/) - cara mengatasi kesalahan `git` yang biasa dijelaskan dalam bahasa Inggeris biasa; lihat juga [Dangit, git!](https://dangitgit.com/) untuk halaman tanpa sumpah.
+- [Atlassian Git Tutorials](https://www.atlassian.com/git/tutorials) - pelbagai tutorial tentang penggunaan `git`.
+- [GitHub Git Cheat Sheet](https://education.github.com/git-cheat-sheet-education.pdf) (PDF)
+- [freeCodeCamp's Wiki on Git Resources](https://forum.freecodecamp.org/t/wiki-git-resources/13136)
+- [GitHub Flow](https://www.youtube.com/watch?v=juLIxo42A_s) (42:06) - bagaimana menarik tarik permintaan diajar oleh GitHub
+- [GitHub Learning Resources](https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github/git-and-github-learning-resources) - Git dan GitHub sumber pelajaran
+- [Pro Git](https://git-scm.com/book/en/v2) - Keseluruhan buku Pro Git, ditulis oleh Scott Chacon dan Ben Straub dan diterbitkan oleh Apress.
+- [Git-it](https://github.com/jlord/git-it-electron) - Langkah demi langkah untuk memuat turun aplikasi Git dalam desktop anda.
+- [Flight Rules for Git](https://github.com/k88hudson/git-flight-rules) - Panduan mengenai apa yang harus dilakukan apabila masalah anda tidak dapat diselesaikan.
+- [Git Guide for Beginners in Spanish](https://platzi.github.io/git-slides/#/) - Panduan lengkap mengenai slaid mengenai git dan GitHub dijelaskan dalam bahasa Sepanyol. Una guía completa de diapositivas sobre git y GitHub explicadas en Español.
+- [Git Kraken](https://www.gitkraken.com/git-client) - Aplikasi desktop `git` visual, lintas platform, dan interaktif untuk kawalan versi.
+- [Git Tips](https://github.com/git-tips/tips) - Koleksi petua dan trik git yang paling biasa digunakan.
+- [Git Best Practices](https://sethrobertson.github.io/GitBestPractices/) - Lakukan Selalunya, Sempurna Kemudian, Terbitkan Sekali: Git Amalan Terbaik
+- [Git Interactive Tutorial](https://learngitbranching.js.org/) - Belajar Git secara visual dan interaktif
```

Comment:
- change `Belajar Git secara visual dan interaktif` to `Belajar Git dengan cara paling visual dan interaktif`